WTWSMS/history/characters/roman.txt

```diff
@@ -3480,6 +3480,18 @@
 		death=yes
 	}
 }
+1310020 = {
+    name="Flavius" #Flavius Ortalinus PLRE III Consul of Lydia
```

Comment:
Isn't Ortalinus the name too?

Reply:
I´m not so sure about it, Ortalinus doesn´t sound as a roman name.

Reply:
Practically everything ending with -us sounds like a Roman name :P

Reply:
What should I do, invent a dynasty or leave it blank?